src/utils.ts

```diff
@@ -3,3 +3,39 @@ import { Point } from '@projectstorm/geometry';
 export function subPoints(point1: Point, point2: Point) {
   return new Point(point1.x - point2.x, point1.y - point2.y);
 }
+
+function makeArray(param: NodeListOf<any>) {
+  return Array.from(param);
+}
+
+export function updateCanvasMouseCursor(cursor = 'text') {
```

Comment:
We probably should store the classnames like '.primary-node' in a variable that we export so we can use it both here and in the application (psyneulink). No need to handle that in the context of this issue, I will open a new issue to handle it later